### ISSUE TITLE
Delete duplicated part on specimen record

### ISSUE THREAD

**Issue by adhornsby** [ArctosDB/arctos#6758]
# Error Text
ERROR_ID | A2DED3BD-1F93-4280-A0A2867E258DD0B6
-- | --
ERROR_TYPE | SQL
ERROR_MESSAGE | ERROR: update or delete on table "specimen_part" violates foreign key constraint "fk_specpartattr_specpart" on table "specimen_part_attribute" Detail: Key (collection_object_id)=(31932672) is still referenced from table "specimen_part_attribute".
ERROR_DETAIL |  
ERROR_SQL | DELETE FROM specimen_part WHERE collection_object_id = 31932672

# Where it happened
Parts tab of https://arctos.database.museum/guid/UMZM:Mamm:477

# Steps to get there
Attempting to delete duplicated part = mandible by hitting the "Delete" button.  Maybe I need to use the "Delete Cascade" button, but I don't know what that means so I wanted to check first!

# Problem
**Community response to describe the problem that caused the error**

# Solution
**Community response with directions for how to correct the problem**

**Comment by dustymc** [comment]
>Maybe I need to use the "Delete Cascade"

Yep - or delete the attributes first, then delete the part. (The button just does that.)

**Comment by adhornsby** [comment]
Thanks @dustymc !
